bc/channel/models.py

```diff
@@ -23,10 +28,19 @@ class Group(AbstractDateTimeModel):
         related_name="groups",
         blank=True,
     )
+    slug = models.SlugField(
+        help_text="A generated path for this item", null=True
+    )
+    overview = models.TextField(
+        help_text="Short description of the purpose of this group", null=True
+    )
```

Comment:
String fields shouldn't ususally be nullable, right? They should be blankable instead?

Reply:
You're right! I'm going to remove the null argument for these fields.